bin/ember-template-lint.js

```diff
@@ -121,6 +121,27 @@ function parseArgv(_argv) {
       }
 
       options.named.filename = filename;
+    } else if (arg === '--quiet') {
+      // TODO: add error handling when named args are out of order
```

Comment:
I should update the comment to be clearer, the specific thing that we should throw on here is if you use `--` **and _then_** use `--quiet` (or any other named argument). This is an error condition because everything after `--` should be filenames to lint.

Reply:
What if the file name **is** `--quiet`? I would expect this

```
ember-template-lint --quiet -- --quiet
```

to be quiet and read a file named `--quiet` Then again we are filtering out by `.hbs` extension so I don't know. Seems like more work for an edge case that is a little nonsensical.

Reply:
> Seems like more work for an edge case that is a little nonsensical.

Yeah, probably. 🤔